src/backend/postmaster/autovacuum.c

```diff
@@ -207,6 +207,11 @@ static int	default_multixact_freeze_table_age;
 /* Memory context for long-lived data */
 static MemoryContext AutovacMemCxt;
 
+/*
+ * Hook for plugins to get control in AutoVacLauncher.
+ */
+AutoVacLauncherMain_hook_type AutoVacLauncherMain_hook = NULL;
```

Comment:
Provide a whitelist/blacklist so that we could control what background worker processes start?

Reply:
Add whitelist function for auxiliary/background process.